src/ert/_c_wrappers/enkf/ert_workflow_list.py

```diff
@@ -110,7 +101,9 @@ def __init__(
                     print(f"WARNING: Unable to create job from {job[ConfigKeys.PATH]}")
                     continue
                 if new_job is not None:
-                    workflow_joblist.addJob(new_job)
+                    self._workflow_jobs[new_job.name()] = workflow_joblist.addJob(
```

Comment:
does actually `addJob` return anything?

Reply:
It should have, but lied about that! Crude fix in next commit. Good catch!